model.py

```diff
@@ -31,25 +31,34 @@ def device(self):
         return next(self.parameters()).device
 
     @argbind.bind
-    def sample(self, batch_size=1, num_steps=100, bias=0.0):
+    def sample(self, batch_size=1, num_steps=100, bias=0.0, stddev=0.1):
         x = torch.zeros(1, batch_size, 3).to(self.device)
         hidden = None
         output = []
         for i in range(num_steps):
             x, hidden = self.step(x, hidden)
+            # Bias is weird now because we have a fixed stddev.  Instead of using bias, we could
+            # just adjust stddev.
+            #
+            # However, in the future, log-stddev will be part of the output of the model, and so it makes sense
+            # to apply bias independently.
+            zeros = torch.zeros(batch_size, 2).to(self.device)
+            std = torch.exp(torch.log(zeros + stddev) - bias)
+            noise = torch.normal(mean=zeros, std=std)
+            x[0, :, 1:3] += noise
```

Comment:
This inplace modification is certainly more efficient than splitting penup and coords, but I think that in general I'd prefer to split for readability.